### PR TITLE
Add config controller

### PR DESCRIPTION
**Pull request by OleksanderSalamatov**
- add full support for config-controller
- fixed minor linter warnings in paymaster predeployed

### CODE REVIEW

**Review by yatsunastya — APPROVED** [review]

**Review by EvgeniyZZ — CHANGES_REQUESTED** [review]
To discuss

File: skale/contracts/config_controller/config_controller.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
+from skale.contracts.base_contract import BaseContract, transaction_method
+from eth_typing import ChecksumAddress
+from web3.contract.contract import ContractFunction
+
+
+class ConfigController(BaseContract):
+    """Config controller contract"""
+
+    def default_admin_role(self) -> hex:
```
Comment (EvgeniyZZ) [inline skale/contracts/config_controller/config_controller.py]: Smart contracts expect to receive bytes. Use bytes type for low level interaction.

File: skale/contracts/config_controller/config_controller.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
+from skale.contracts.base_contract import BaseContract, transaction_method
+from eth_typing import ChecksumAddress
```
Comment (EvgeniyZZ) [inline skale/contracts/config_controller/config_controller.py:2]: Use:
from eth_typing import ChecksumAddress, HexStr
insted of -> hex: 
correct -> HexStr

File: skale/contracts/config_controller/config_controller.py [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+from web3.contract.contract import ContractFunction
```
Comment (EvgeniyZZ) [inline skale/contracts/config_controller/config_controller.py]: Use TxRes

**Review by EvgeniyZZ — APPROVED** [review]

**Review by EvgeniyZZ — APPROVED** [review]

**Pull request merged**